### ISSUE TITLE
代码中在计算loss时的lowface_mask实际上是上半脸区？

### ISSUE THREAD

**Issue by int33** [ZiqiaoPeng/SyncTalk#156]
# aud att loss (regions out of face should be static)
        if self.opt.amb_aud_loss and not self.opt.torso:
            ambient_aud = outputs['ambient_aud']
            loss_amb_aud = (ambient_aud * (~lowface_mask.view(-1))).mean()
            # gradually increase it
            lambda_amb = step_factor * self.opt.lambda_amb 
            loss += lambda_amb * loss_amb_aud

        # eye att loss
        if self.opt.amb_eye_loss and not self.opt.torso:
            ambient_eye = outputs['ambient_eye']
            loss_cross = ((ambient_eye)*(lowface_mask.view(-1))).mean()#注意lowface_mask
            lambda_amb = step_factor * self.opt.lambda_amb
            loss += lambda_amb * loss_cross`

**Comment by int33** [comment]
上部分代码来自SyncTalk工作中utils.py大概885行左右，eye_att_loss应该对应的是上半脸区的表情部分，所以lowface_mask说是lowface实际对应的上半脸区？

**Comment by ZiqiaoPeng** [comment]
lowface_mask是下半张脸。aud att loss是为了让"~lowface_mask"(即下脸以外的区域注意力最小)，eye att loss是为了让"lowface_mask"(下脸区域注意力最小)。

**Comment by int33** [comment]
原来是这样，非常感谢
